docs/reference/functional-testing.md

```diff
@@ -413,32 +395,51 @@ When encountering a failed test in CircleCI, follow these systematic steps for e
+1. Do not comment-out test cases:
+   Instead, use `test.fixme()` and `test.skip()` annotations. This allows for better metric calculation and Static Analysis/IDE feedback.
+   - Reference a JIRA ticket in the annotations whenever possible
+   - `test.fixme()` is preferred for tests in progress or tests requiring maintenance
+   - `test.skip()` is preferred for project or feature flag constraints
+    ```ts
+    // Project
+    test.skip(
+      project.name === "production",
+      "The test case is unsupported in production."
+    );
 
-2. Exclude Partially Implemented Tests from CI Execution:
-Partially implemented tests should not be executed in the Continuous Integration (CI) pipeline. Only fully functional and thoroughly validated tests should be included in CI runs to maintain the reliability of our testing infrastructure.
+    // Feature Flag
+    test.skip(config.showReactApp.signInRoutes === true, "FXA-9410");
+    ```
 
-3. Exercise Prudent Judgment with 3rd Party UIs & Flows:
-When interacting with third-party User Interfaces (UIs) and workflows, exercise caution and discretion. Ensure that test scenarios align with expected behaviors and handle potential variations in third-party components.
+2. Exclude partially implemented tests from CI execution:
+   Partial implementation can lead to miscommunication in the form of false positives
 
-4. Prefer Fixtures Over Setup & Teardown Lifecycle Methods:
-Favor the use of fixtures over traditional setup and teardown lifecycle methods. Fixtures provide a more modular and reusable approach, enhancing the maintainability and scalability of our test suites.
+3. Exercise judgment with 3rd party UIs & flows:
+   Generally speaking automating 3rd party interfaces is discouraged, since it can introduce brittleness and be the cause of false negatives
 
-5. Avoid Deprecated Playwright Functions:
-Stay vigilant and avoid the use of deprecated functions in Playwright. Regularly consult the Playwright documentation for updates and transition to recommended alternatives to ensure compatibility and future-proofing of our test scripts.
+4. Prefer Fixtures over setup & teardown lifecycle methods:
+   As recommended by Playwright, fixtures best promote reusability and facilitate test concurrency
 
-> These guidelines are pivotal for promoting consistency, reliability, and efficiency within our testing practices. Adherence to these best practices will contribute to the overall success and resilience of our test automation ecosystem.
+5. Avoid Deprecated Playwright Functions:
+   Stay vigilant and avoid the use of deprecated functions in Playwright. Regularly consult the Playwright documentation for updates and transition to recommended alternatives to ensure compatibility and future-proofing of our test scripts
```

Comment:
Not 100% sold on this/my suggestion (since it doesn't point to some magical definitive list of deprecated APIs), but figure it might be nice to add a free link to the Playwright docs; which just _happen_ to be their official "best practices" docs.

```suggestion
   Stay vigilant and avoid the use of deprecated functions in Playwright. Regularly consult the [Playwright documentation](https://playwright.dev/docs/best-practices) for updates and transition to recommended alternatives to ensure compatibility and future-proofing of our test scripts
```

Reply:
Your feedback actually makes me think we should remove the guideline. This is probably general to all software development and probably doesn't need to be stated.